src/app/shared/input/input.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="form-row">
+  <div class="col-md mb-3">
+    <label>{{ label }}</label>
+    <input
```

Comment:
fajny pomysl - ale co jesli
 - bede chcial uzyc textarea?
- bede chcial pozniej gdzies uzyc tej kontrolki i zapiac sie na event blur czy focus?

polecam skorzystanie z projekcji i ng-content :) wtedy mozesz zrobic cos takiego <my-input><input..... > </my-input> 
Przy dobrej kombinacji jestes nawet w stanie wyciagnac formControl z tego inputa ale to troche wyzsza szkola jazdy :)

Reply:
ok, poprawione